src/LaunchDarkly.ServerSdk.Redis/RedisBigSegmentStoreImpl.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LaunchDarkly.Sdk.Server.Interfaces;
+using LaunchDarkly.Logging;
+using StackExchange.Redis;
+
+using static LaunchDarkly.Sdk.Server.Interfaces.BigSegmentStoreTypes;
+
+namespace LaunchDarkly.Sdk.Server.Integrations
+{
+    internal sealed class RedisBigSegmentStoreImpl : RedisStoreImplBase, IBigSegmentStore
+    {
+        private readonly string _syncTimeKey;
+        private readonly string _includedKeyPrefix;
+        private readonly string _excludedKeyPrefix;
+
+        internal RedisBigSegmentStoreImpl(
+            ConfigurationOptions redisConfig,
+            string prefix,
+            Logger log
+            ) : base(redisConfig, prefix, log)
+        {
+            _syncTimeKey = prefix + ":big_segments_synchronized_on";
+            _includedKeyPrefix = prefix + ":big_segment_include:";
+            _excludedKeyPrefix = prefix + ":big_segment_exclude:";
+        }
+
+        public async Task<IMembership> GetMembershipAsync(string userHash)
+        {
+            var db = _redis.GetDatabase();
+
+            var includedRefs = await db.SetMembersAsync(_includedKeyPrefix + userHash);
+            var excludedRefs = await db.SetMembersAsync(_excludedKeyPrefix + userHash);
+
+            return NewMembershipFromSegmentRefs(RedisValuesToStrings(includedRefs),
+                RedisValuesToStrings(excludedRefs));
+            throw new NotImplementedException();
```

Comment:
Looks like a stub implementation didn't get deleted here.

Reply:
🤦 ... I guess I've gotten overly accustomed to the Go compiler's behavior of disallowing unreachable code, which .NET apparently doesn't care about.